client/my-sites/plans-features-main/plan-type-selector.tsx

```diff
@@ -146,49 +154,46 @@ export const IntervalTypeToggle: React.FunctionComponent< IntervalTypeProps > =
 	const isDomainAndPlanPackageFlow = new URLSearchParams( window.location.search ).get(
 		'domainAndPlanPackage'
 	);
+	const shouldHideMonthly = shortestTermOption === '1year';
```

Comment:
I think this is creating some additional knowledge here that isn't necessary, at least not currently i.e. that the "shortest term option" being "1year" implies that "monthly" should be hidden. It feels like unnecessary complexity. Why not just keep it simple for now e.g. `showBiannualToggle`? 

I can envision something like `shortestTermOption` being useful for showing more than 2 values e.g. everything from `shortestTermOption` and above, but this is not the case with the rest of the logic. 🤔

Reply:
@chriskmnds good call, I'll switch it to that approach. Thanks for the review!